File: Source/DroolTool.Models/DataTransferObjects/WaterAccountsChartDto.cs
namespace DroolTool.API.Controllers
{
    public class WaterAccountsChartDto
    {
        public int MetricMonth { get; set; }
        public int MetricYear { get; set; }
        public double? ResidentialWaterAccounts { get; set; }
        public double? HOAWaterAccounts { get; set; }
        public double? CommercialWaterAccounts { get; set; }
        public double? MunicipalWaterAccounts { get; set; }
    }
}

File: DroolTool.Web/src/app/shared/models/water-accounts-chart-dto.ts
export class WaterAccountsChartDto {
    public MetricYear: number;
    public MetricMonth: number;
    public ResidentialWaterAccounts?: number;
    public HOAWaterAccounts?: number;
    public CommercialWaterAccounts?: number;
    public MunicipalWaterAccounts?: number;
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}